src/pages/singlePoll.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="h-screen m-auto flex justify-center" v-if="isLoading">
```

Comment:
create a loader component for it, and use it every where @ShivanshSinghnnovative